bin/run_tests.sh

```diff
@@ -28,6 +28,21 @@ do
   cd $base
 done
 
+for shared_library in $(ls -d tasks/shared_libraries/* | grep recovery)
```

Comment:
The ` | grep recovery` is not needed. There is currently only one directory under `tasks/shared_libraries`  and no need to filter.

I would also add a comment above this loop explaining what it does/is for